drivers/display/display_mcux_dcnano_lcdif.c

```diff
@@ -278,6 +297,29 @@ static DEVICE_API(display, mcux_dcnano_lcdif_api) = {
 #define MCUX_DCNANO_LCDIF_FRAMEBUFFER(n) mcux_dcnano_lcdif_frame_buffer_##n
 #endif
 
+#if DT_ENUM_IDX_OR(DT_NODELABEL(lcdif), version, 0) == 1
+#define MCUX_DCNANO_LCDIF_FB_CONFIG(n)					\
```

Comment:
Nit here- please align the `\` backslashes on each line, to remain stylistically consistent with the rest of the file.

Reply:
Thanks, fixed.